src/main/java/org/jabref/gui/groups/GroupDialog.java

```diff
@@ -566,7 +571,9 @@ private void updateComponents() {
             s1 = searchGroupSearchExpression.getText().trim();
             okEnabled = okEnabled & !s1.isEmpty();
             if (okEnabled) {
-                setDescription(new SearchQuery(s1, isCaseSensitive(), isRegex()).getDescription());
+                SearchQuery searchQuery = new SearchQuery(s1, isCaseSensitive(), isRegex());
+                setDescription(fromTextFlowToHTMLString(SearchDescribers.getSearchDescriberFor(searchQuery.getRule(),
```

Comment:
I think it makes sense to introduce an overload of `getSearchDescriberFor` that accepts a `SearchQuery` (or even change the signature directly).